py/util.py

```diff
@@ -278,6 +280,39 @@ def wait_for_deployment(api_client, namespace, name):
       "Timeout waiting for deployment {0} in namespace {1}".format(
       name, namespace))
 
+def wait_for_statefulset(api_client, namespace, name):
```

Comment:
I have this feeling that _wait for <resource type> to be <ready|created|deleted>_ is commonly used and repeatedly reimplemented in both Python and Golang

Reply:
Probably. Do you know if there is a common library somewhere?